colibris/skeleton/__packagename__/routes.py

```diff
@@ -10,9 +10,9 @@
 ROUTES = [
     (r'/', views.HomeView),
     (r'/health', views.HealthView),
-    (r'/users/me', views.MeView),
-    (r'/users', views.UsersView),
-    (r'/users/{id:\d+}', views.UserView)
+    (r'/api/users/me', views.MeView),
```

Comment:
Why do we want this change? Also, the current skeleton routes don't scope the API calls under `/api`.